lib/phoneNumber.js

```diff
@@ -86,6 +88,18 @@ class PhoneNumber {
     return false;
   }
 
+  format(number, iso2) {
+    const formatter = new asYouTypeFormatter(iso2)
+    let formatted;
+
+    number.replace(/-/g, '')
+      .replace(/ /g, '')
+      .split('')
+      .forEach(n => formatted = formatter.inputDigit(n));
+
```

Comment:
is not the case of this advice?

https://github.com/thegamenicorus/react-native-phone-input/pull/70#issuecomment-457857601